lib/matplotlib/table.py

```diff
@@ -118,7 +135,7 @@ def _set_text_position(self, renderer):
         l, b, w, h = bbox.bounds
 
         # draw in center vertically
-        self._text.set_verticalalignment('center')
+        self._text.set_verticalalignment('center_baseline')
```

Comment:
Can you point me at where exactly this "API changes" thing is that needs a note?